ts/theme/variables.ts

```diff
@@ -121,6 +124,12 @@ const customVariables = Object.assign(materialVariables, {
   // H1
   h1Color: "#17324D",
   h1FontWeight: "700" as FontWeight,
+  get h1FontSize(): number {
+    return this.fontSize6;
```

Comment:
why `h1` correspond to `fontSize6` ?!?

Reply:
Should I hardcode the font-size value?

Reply:
what I mean is: do we have a convention? it seems on one hand we have big header == lower number (`h1`) and on the other big header == high number (`fontSize6`) - cc @fpersico

Reply:
Furthermore, I double checked with Invision project: I found out that the screen titles don't have the correct font-size (28). We should create a new component for this type of titles, with che correct font-size (40).